package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "build": "tsc && yarn copy-types",
     "watch": "tsc --watch",
+    "secret": "echo $EXAMPLE_SECRET",
```

Comment:
It looks like github actions did not run which is good.

But neither did Circle CI? 🤔